random-source/random-source.cabal

```diff
@@ -82,7 +82,7 @@ Library
   
   build-depends:        flexible-defaults >= 0.0.0.2,
                         mersenne-random-pure64,
-                        random,
+                        random == 1.2.0,
```

Comment:
`==` should be `>=`, or maybe we should have `random == 1.2.*` instead (see previous comment)

Reply:
Done